iroha/src/torii.rs

```diff
@@ -127,7 +138,14 @@ async fn handle_request(state: State<ToriiState>, request: Request) -> Result<Re
                 Ok(Response::InternalError)
             }
         },
-        uri::HEALTH_URI => Ok(Response::Ok(crate::maintenance::Health::Healthy.into())),
+        uri::HEALTH_URI => Ok(Response::Ok(Health::Healthy.into())),
+        uri::METRICS_URI => match state.read().await.system.read().await.scrape_metrics() {
+            Ok(metrics) => Ok(Response::Ok(metrics.into())),
+            Err(e) => {
+                eprintln!("Failed to decode peer message: {}", e);
```

Comment:
Probably a different error message should be used?

Reply:
Much thanks